apps/cowswap-frontend/src/modules/twap/hooks/useTwapOrdersAuthMulticall.ts

```diff
@@ -20,7 +20,7 @@ export function useTwapOrdersAuthMulticall(
     return ordersInfo.map(({ id }) => [safeAddress, id])
   }, [safeAddress, ordersInfo])
 
-  const results = useSingleContractMultipleData<boolean>(
+  const { data: loadedResults, isLoading } = useSingleContractMultipleData<[boolean]>(
```

Comment:
#3809 fix

Reply:
Why change the type from `boolean` to `[boolean]` ?
Was that also wrong?

Reply:
Yes, this is a part of https://github.com/cowprotocol/cowswap/pull/3831#discussion_r1486105396